src/write_outputs/write_costs.jl

```diff
@@ -11,75 +11,90 @@ function write_costs(path::AbstractString, inputs::Dict, setup::Dict, EP::Model)
 	T = inputs["T"]     # Number of time steps (hours)
 	VRE_STOR = inputs["VRE_STOR"]
 	
-	cost_list = ["cTotal", "cFix", "cVar", "cNSE", "cStart", "cUnmetRsv", "cNetworkExp", "cUnmetPolicyPenalty"]
+	cost_list = ["cTotal", "cFix", "cVar", "cFuel" ,"cNSE", "cStart", "cStartFuel", "cUnmetRsv", "cNetworkExp", "cUnmetPolicyPenalty", "cCO2"]
```

Comment:
Is there a reason why it's important to separate fuel costs from start-fuel costs?

Reply:
I think it is more clear to have it broken out. Also why is there a cCO2 if there is no CO2 tax module (yet)?

Reply:
I asked Jesse today and he suggested separating cVar and cFuel, but keeping startup cost and starup fuel together. So I will combine startup and startfuel together as both are tiny values..